pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -162,6 +162,9 @@ func (dswp *desiredStateOfWorldPopulator) findAndRemoveDeletedPods() {
 			}
 			// Skip non-memory backed volumes belonging to terminated pods
 			volume := volumeToMount.VolumeSpec.Volume
+			if volume == nil {
+				continue
+			}
 			if (volume.EmptyDir == nil || volume.EmptyDir.Medium != v1.StorageMediumMemory) &&
```

Comment:
> Also - it may be good idea to cover this with some test? Not sure how easy that will be though.

I'll try to write a test as part of my follow-on PR.